src/collection/resources/_methods.js

```diff
@@ -1,12 +1,12 @@
 module.exports = (ObjectClass) => ({
   getAll({
-    page, size, urlParams, json = false,
+    page, size, urlParams, tags, json = false,
   } = {}, done) {
     return this.paginate({
       page,
       size,
       type: 'get',
-      urlParams,
+      urlParams: { ...urlParams, tags: tags ? tags.toString() : undefined },
```

Comment:
you could have used `...(tags && { tags: tags.toString() })` and avoid the `undefined`